### PR TITLE
MAVProxy: Updated install guide

### PR DESCRIPTION
**Pull request by stephendade**
Clearer install instructions for MAVProxy. I've also tested the Linux ``apt-get``'s to ensure there's no missing packages

### CODE REVIEW

File: mavproxy/source/docs/getting_started/download_and_installation.rst [hunk 1]
```diff
@@ -37,45 +40,41 @@ within Windows. It generally provides better stability than the native Windows i
...
-    
-For Fedora based systems:
+
+For Python 2 on Debian based systems (including Ubuntu, WSL, Raspian):
 
 .. code:: bash
 
-    sudo dnf install python3-devel python3-opencv wxPython python3-pip python3-matplotlib pygame python3-lxml python3-yaml redhat-rpm-config
-    pip3 install mavproxy --user
+    sudo apt-get install python-dev python-opencv python-wxgtk4.0 python-pip python-matplotlib python-lxml python-pygame
```
Comment (Hwurzburg) [inline mavproxy/source/docs/getting_started/download_and_installation.rst:58]: I thought pip3 is only a Python 3 package

File: mavproxy/source/docs/getting_started/download_and_installation.rst [hunk 2]
```diff
@@ -37,45 +40,41 @@ within Windows. It generally provides better stability than the native Windows i
...
+    pip3 install PyYAML mavproxy --user
```
Comment (Hwurzburg) [inline mavproxy/source/docs/getting_started/download_and_installation.rst:59]: I thought pip3 was python 3..why under a python 2 instal?

File: mavproxy/source/docs/getting_started/download_and_installation.rst [hunk 3]
```diff
@@ -37,45 +40,41 @@ within Windows. It generally provides better stability than the native Windows i
...
-For Raspian / Raspberry Pi OS:
+    
+For Python 3 on Fedora based systems:
 
 .. code:: bash
 
-    sudo apt-get install python3-dev python3-opencv python3-wxgtk4.0 python3-pip python3-matplotlib python3-pygame python3-lxml python3-yaml libxml2-dev
-    pip3 install mavproxy --user
+    sudo dnf install python3-devel python3-opencv python3-wxpython4 python3-pip python3-matplotlib python3-pygame python3-lxml python3-yaml redhat-rpm-config
+    pip install PyYAML mavproxy --user
```
Comment (Hwurzburg) [inline mavproxy/source/docs/getting_started/download_and_installation.rst:67]: same question here but re Python2....pip and not pip3?

File: mavproxy/source/docs/getting_started/download_and_installation.rst (hunk 2 above)
Comment (stephendade) [reply mavproxy/source/docs/getting_started/download_and_installation.rst]: Correct. Copy-pasting to wrong section of text. Now fixed.

File: mavproxy/source/docs/getting_started/download_and_installation.rst (hunk 3 above)
Comment (stephendade) [reply mavproxy/source/docs/getting_started/download_and_installation.rst]: Correct. Copy-pasting to wrong section of text. Now fixed.

**Comment by Hwurzburg** [comment]
Thanks! Merged

**Comment by Hwurzburg** [comment]
When I get home..can't merge via phone

**Pull request merged**